kentikapi/client.go

```diff
@@ -132,25 +122,34 @@ func (c *Config) FillDefaults() {
 		c.APIURL = APIURLUS
 	}
 
-	if c.CloudExportHostPort == "" {
-		c.CloudExportHostPort = cloudExportHostPort
-	}
-
-	if c.SyntheticsHostPort == "" {
-		c.SyntheticsHostPort = syntheticsHostPort
-	}
-
 	if c.Timeout == nil {
 		c.Timeout = pointer.ToDuration(defaultTimeout)
 	}
 
 	c.RetryCfg.FillDefaults()
 }
 
-func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error) {
+func parseKentikURL(apiURL *string) (string, bool) {
+	tls := false
+	u, err := url.Parse(*apiURL)
+	if err == nil {
+		*apiURL += "/api/v5"
```

Comment:
Does this mean we both update input apiURL and return output grpc?

Reply:
Yes if input string is a valid url. This is a bit strange logic but needed for tests.  
Normally when user pass a valid url like https://api.kentik.com we want to have https://api.kentik.com/api/v5 and grpc.api.kentik.com:443 url parameters.
When we running test, input string will be local host with port like: 127.0.0.1:3869 This is not a valid url and should be api endpoint without modifications.

Reply:
All 3 done